### ISSUE TITLE
Optimalisere kommunikasjon mellom dataset-catalog og dataset-catalog-gui

### ISSUE THREAD

**Issue by matsjs** [Informasjonsforvaltning/dataset-catalog#38]
Front end krever at datasett og kataloger er inne i objekter ved navn "_embedded" som igjen inneholder liste med " catalogs" eller "datasets", dette kunne bare vært en liste med objektene

**Comment by KjerstiSteien** [comment]
@NilsOveTen @hegeaal vil denne saken være relevant når vi har flyttet skjema for datasett over i ny løsning?

**Comment by NilsOveTen** [comment]
@KjerstiSteien Ja, og det er et veldig greit tidspunkt å gjennomføre den refaktoreringen.

**Comment by KjerstiSteien** [comment]
Da løfter vi denne frem i det arbeidet?

**Comment by NilsOveTen** [comment]
Jeg synes vi burde løfte frem en litt større refaktorering av dataset-catalog, dataservice-catalog og concept-catalog, men ikke feil om vi starter her

**Comment by KjerstiSteien** [comment]
@hegeaal @NilsOveTen ble det gjort noe med dette i forbindelse med nytt skjema i datasett?

**Comment by NilsOveTen** [comment]
@hegeaal har en liste over potensielle forbedringer, men har utsatt til etter nytt skjema er i bruk. Å gjøre den type endringer når vi har 3 forskjellige frontender å ta hensyn til er knotete
